disco_aws_automation/disco_aws.py

```diff
@@ -422,7 +422,7 @@ def provision(self, ami, hostclass=None,
             group_name=group_name
         )
 
-        self.create_scaling_schedule(hostclass, min_size, desired_size, max_size)
+        self.create_scaling_schedule(min_size, desired_size, max_size, group_name=group.name)
```

Comment:
i'm going to assume that `group.name` is not a typo and there's a `group` object somewhere before that call?